### PR TITLE
More of opflow sensor support

### PR DESCRIPTION
**Pull request by digitalentity**
1. Refactor optical flow sensor support
2. Add support for Cheerson CXOF flow sensor (via UART)
3. Refactor VEL_XY controller to have more P-authority

WARNING! After this PR pilots will need to reduce their VEL_XY P-gain by factor of 5 (five)

### CODE REVIEW

File: src/main/fc/fc_msp.c [hunk 1]
```diff
@@ -485,6 +485,22 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
...
 
+    case MSP2_INAV_OPTICAL_FLOW:
+#ifdef USE_OPTICAL_FLOW
+        sbufWriteU8(dst, opflow.rawQuality);
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.flowRate[X]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.flowRate[Y]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.bodyRate[X]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.bodyRate[Y]));
+#else
+        sbufWriteU8(dst, opflow.rawQuality);
```
Comment (martinbudden) [inline src/main/fc/fc_msp.c]: I can't see a difference between the `#if` and the `#else` clause.

Comment (digitalentity) [reply src/main/fc/fc_msp.c]: Oops!

**Comment by Pairan** [comment]
Just ordering one to put it on the versacopter - then I will be ready for testing with you

**Comment by Pairan** [comment]
CXOF is here and ready for testing with a  Versacopter and `OMNIBUS`

**Pull request merged**